Podfile

```diff
@@ -160,7 +160,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.47.0'
+    gutenberg :tag => 'v1.47.2'
```

Comment:
I was a bit surprised to see this jump from `v1.47.0` to `v1.47.2` (skipping the `1.47.1` version), but it looks like this middle version [was Android-specific](https://github.com/WordPress/gutenberg/pull/29626), so WPiOS was not updated.
cc @bjtitus @fluiddot

Reply:
Yeah exactly, the version `1.47.1` was a hotfix that was only released on Android. I'm going to add `(only Android)` in the related PRs of `1.47.1` to avoid confusion.

Reply:
Thanks for confirming @fluiddot. Incidentally this 1.47.2 hotfix is iOS-specific, so we didn't update WPAndroid here.